plugins/tileacq.py

```diff
@@ -245,16 +381,23 @@ def _move_to_tile(self, idx, orig_pos, tile_size):
         # |
         # --->-->-->--Z
         overlap = 1 - self.overlap.value / 100
-        tile_pos = (orig_pos["x"] + idx[0] * tile_size[0] * overlap,
-                    orig_pos["y"] - idx[1] * tile_size[1] * overlap)
+        # don't move on the axis that is not supposed to have changed
+        main_data = self.main_app.main_data
+        prev_pos = main_data.stage.position.value
+        if numpy.argmax(abs(numpy.subtract(idx, prev_idx))) == 0:
```

Comment:
It's better to not even ask to move on the axes which are identical. => build the move dict on the fly.
So, something in this vain:
```
m = {}
idx_change = numpy.subtract(idx, prev_idx)
if idx_change[0]:
   m["x"] = orig_pos["x"] + idx[0] * tile_size[0] * overlap
if idx_change[1]:
   m["y"] = orig_pos["y"] + idx[1] * tile_size[1] * overlap
:
moveAbs(m)
```